src/upload/UploadedFile.stories.tsx

```diff
@@ -21,6 +22,16 @@ export const withoutCancelButton = () => <Paper>
     </Grid>
 </Paper>;
 
+withoutCancelButton.story = {
```

Comment:
It is not needed to test for this, screenshot testing already takes this into account